electron_app/src/ipc/mailbox.js

```diff
@@ -82,7 +82,8 @@ ipc.answerRenderer(
           openFolderWhenDone: !isInlineImage
         }
       );
-      return downloadedItem.getSavePath();
+      const newFilePath = downloadedItem.getSavePath();
+      return { filePath: newFilePath, filename };
     } catch (e) {
       if (!isInlineImage)
         return mailboxWindow.send('display-message-error-download');
```

Comment:
Add a empty object in catch for preventing crash `Can not read filename of undefined`